keyboards/etiennecollin/wave/readme.md

```diff
@@ -0,0 +1,27 @@
+# Wave
+
+![Wave pcb](https://raw.githubusercontent.com/etiennecollin/wave/main/resources/images/wave_pcb_rounded.svg)
```

Comment:
Please rehost this on Imgur.

Reply:
Sure, but I thought the checklist only **suggested** imgur:

> - images should be uploaded to an external image hosting service, **such as** [imgur](https://imgur.com/).
> - **if** imgur is used, images should be resized appropriately: append “h” to the image url i.e. https://i.imgur.com/vqgE7Ok.jpg becomes https://i.imgur.com/vqgE7Okh.jpg
> - image links should link directly to the image, not a “preview” – i.e. https://imgur.com/vqgE7Ok should be https://i.imgur.com/vqgE7Okh.jpg **when using imgur**
>
> [Source](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)

I wanted to host it on GitHub because it prevents the duplication of files which are already used there and makes sure that only one file has to be updated whenever there are changes. Is it really mandatory to use imgur? The link provided is not a preview and links directly to the image.

Reply:
Github links are prone to changing whether it be the image/repository path/name is changed, Github changing their link format, etc. This is why Imgur is suggested. If you have another hosting service that can provide the same reliability, that's fine.

Reply:
Good to know! I will change it.